qiita_ware/processing_pipeline.py

```diff
@@ -22,51 +22,56 @@
 
 def _get_qiime_minimal_mapping(prep_template, out_dir):
     """Generates a minimal QIIME-compliant mapping file for split libraries
-
```

Comment:
No idea, I think it has been a copy-paste issue from my other PR.... adding them back.